src/streamer.ts

```diff
@@ -6,7 +6,7 @@ import { sleep } from "./utils";
 async function* batchStream(
   config: LakeConfig
 ): AsyncIterableIterator<Promise<StreamerMessage>[]> {
-  const s3Client = new S3Client({ region: config.s3RegionName });
+  const s3Client = new S3Client({ region: config.s3RegionName, endpoint: config.s3Endpoint,forcePathStyle: true });
```

Comment:
Could you, please, leave a comment as to the need for `forcePathStyle`?

Reply:
the parameter force the path in the format s3endpoint/region/bucket rather than bucket.region.endpoint.
When running local S3 people usually don't configure subdomains. here is the aws documentation: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/s3clientconfig.html#forcepathstyle

If you want to keep the original format I can make this parameter configurable in LakeConfig object.

Reply:
Let's make it configurable, so native AWS S3 benefit from using domains

Reply:
ok, just added it